src/app/shared/stache/modules/layout/layout-sidebar.component.html

```diff
@@ -24,7 +24,7 @@
       </stache-column>
       <stache-column screenSmall="4" screenMedium="3" screenLarge="2">
 
-        <stache-sidebar [routes]="sidebarRoutes">
+        <stache-sidebar stickToTop [routes]="sidebarRoutes">
```

Comment:
Needs to be prefixed with `stache`: `stacheStickToTop`. Or, `stacheAffix`, something like that...